guides/common/Makefile

```diff
@@ -48,6 +50,7 @@ open-pdf: pdf
 	${BROWSER_OPEN} "$(realpath $(ROOTDIR)/$(DEST_PDF))"
 
 $(DEST_DIR)/$(BUILD).css: $(COMMON_DIR)/$(BUILD).css
+	bundle exec compass compile --force -c $(COMMON_DIR)/sass_config.rb --sass-dir $(COMMON_DIR)/sass --css-dir $(COMMON_DIR);
 	cp $< $@
```

Comment:
This is not correct, in Makefile the syntax is:

```
target_file: sourcefile1 sourcefile2 ...etc
  command to run
```

Previously the css target was a simple copy via `cp`. You want to change that to run a different command (`compass` in this case). You want it to create `$(DEST_DIR)/$(BUILD).css`, I am not sure how to do this tho, the utility do not appear to have any `--output` argument to specify where to store the result. Do not store anything in the common directory, use the build directory for the output.

It is very important that the target file and source(s) are valid paths, Makefile is basically a dependency solving tool. What you have here does not work.